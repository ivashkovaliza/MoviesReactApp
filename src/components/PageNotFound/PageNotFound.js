import React from "react";
import './PageNotFound.scss';
import {Link} from "react-router-dom";

const PageNotFound = () => {
  return (
    <div className={'error__wrapper'}>
      <h1 className={'error__msg'}>404 Page Not Found</h1>
      <Link to={'/'} className={'error__link'}>Return to Home page</Link>
    </div>
  );
};

export default PageNotFound;