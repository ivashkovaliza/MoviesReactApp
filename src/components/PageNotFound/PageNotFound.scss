@import '../../styles/_colors';

.error {
  &__wrapper{
    width: 100%;
    height: 100vh;
    background-color: $dark-grey;
    display: flex;
    flex-direction: column;
    justify-content: center;
    align-items: center;
    text-align: center;
  }

  &__msg {
    font-size: 60px;
    text-transform: uppercase;
    font-weight: 200;
  }

  &__link {
    color: $salmon;

    &:hover {
      color: $light-salmon
    }
  }
}
