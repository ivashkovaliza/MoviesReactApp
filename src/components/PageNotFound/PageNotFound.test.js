import React from 'react';
import PageNotFound from './PageNotFound';

describe('PageNotFound', () => {
  test('renders correctly', () => {
    const component = shallow(
      <PageNotFound />
    );
    expect(component).toMatchSnapshot();
  });
});
