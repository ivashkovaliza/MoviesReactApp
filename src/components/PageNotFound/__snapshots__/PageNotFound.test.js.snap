// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFound renders correctly 1`] = `
<div
  className="error__wrapper"
>
  <h1
    className="error__msg"
  >
    404 Page Not Found
  </h1>
  <Link
    className="error__link"
    to="/"
  >
    Return to Home page
  </Link>
</div>
`;
